aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -492,6 +492,30 @@ The ASP.NET Core Module installer runs with the privileges of the **SYSTEM** acc
 1. Export the updated *applicationHost.config* file to the share.
 1. Re-enable the IIS Shared Configuration.
 
+::: moniker range=">= aspnetcore-2.2"
+
+## Application Initialization
+
+App initialization is achieved for apps that use the [in-process hosting model](xref:fundamentals/servers/index#in-process-hosting-model) with [IIS Application Initialization](/iis/get-started/whats-new-in-iis-8/iis-80-application-initialization):
+
+1. Confirm that the IIS Application Initialization role feature in enabled:
+   * On Windows 7 or later: Navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen). Open **Internet Information Services** > **World Wide Web Services** > **Application Development Features**. Select the check box for **Application Initialization**.
+   * On Windows Server 2008 R2 or later, open the **Add Roles and Features Wizard**. When you reach the **Select role services** panel, open the **Application Development** node and select the **Application Initialization** check box.
+1. In IIS Manager, select **Application Pools** in the **Connections** panel.
+1. Select the app's app pool in the list.
+1. Select **Advanced Settings** under **Edit Application Pool** in the **Actions** panel.
+1. Set **Start Mode** to **AlwaysRunning**.
```

Comment:
Can you confirm that you need to set AlwaysRunning here? I can't remember if we can just do Preload Enabled = True or not.

Reply:
I'll check ... I didn't check preload + "on demand" for the start mode.